docs/running/wdl.rst

```diff
@@ -3,15 +3,121 @@
+to use other Toil-supported batch systems, such as Kubernetes:
+
+``toil-wdl-runner --jobStore aws:us-west-2:wdl-job-store --batchSystem kubernetes myWorkflow.wdl --inputs myWorkflow_inputs.json``
+
+For Toil, the ``--inputs`` is optiona, and inputs can be passed as a positional
+argument:
+
+``toil-wdl-runner myWorkflow.wdl myWorkflow_inputs.json``
+
+You can also run workflows from URLs. For example, to run the MiniWDL self test
+workflow, you can do:
+
+``toil-wdl-runner https://raw.githubusercontent.com/DataBiosphere/toil/36b54c45e8554ded5093bcdd03edb2f6b0d93887/src/toil/test/wdl/miniwdl_self_test/self_test.wdl https://raw.githubusercontent.com/DataBiosphere/toil/36b54c45e8554ded5093bcdd03edb2f6b0d93887/src/toil/test/wdl/miniwdl_self_test/inputs.json``
+
+Toil WDL Runner Options
+-----------------------
+
+**'-\\-jobStore'**: Specifies where to keep the Toil state information while
+running the workflow. Must be accessible from all machines.
+
+**'-o'** or **'-\\-outputDirectory'**: Specifies the output folder to save
+workflow output files in. Defaults to a new directory in the current directory.
+
+**'-m'** or **'-\\-outputFile'**: Specifies a JSON file to save workflow output
+values to. Defaults to standard output.
+
+**'-i'** or **'-\\-input'**: Alternative to the positional argument for the
+input JSON file, for compatibility with other WDL runners.
+
+**'-\\-outputDialect'**: Specifies an output format dialect. Can be
+``cromwell`` to just return the workflow's output values as JSON or ``miniwdl``
+to nest that under an ``outputs`` key and includes a ``dir`` key.
+
+Any number of other Toil options may also be specified. For defined Toil options,
+see the documentation:
+http://toil.readthedocs.io/en/latest/running/cliOptions.html
+
+Running WDL within Toil Scripts
+-------------------------------
+
+Currently, the recommended way to run WDL code from within a Toil Python
+workflow is to call out to a WDL runner like ``toil-wdl-runner`` or Cromwell,
+within a Toil job.
+
+.. note::
+        A ``cromwell.jar`` file is needed in order to run a WDL workflow with Cromwell.
+
+This is not very efficient, but for some use cases, such as running one process
+on multiple files, it may be useful. For example, if you want to run a WDL
+workflow with 3 JSON files specifying different samples inputs, it could look
+something like:
+
+.. literalinclude:: ../../src/toil/test/docs/scripts/tutorial_wdlexample.py
+
+WDL Specifications
+------------------
+WDL language specifications can be found here: https://github.com/broadinstitute/wdl/blob/develop/SPEC.md
+
+Toil is not yet fully conformant with the WDL specification, but it inherits most of the functionality of MiniWDL_.
+
+.. _MiniWDL: https://github.com/chanzuckerberg/miniwdl/#miniwdl
+
+Using the Old WDL Compiler
+--------------------------
+
+Up through Toil 5.9.2, ``toil-wdl-runner`` worked by compiling the WDL code to
```

Comment:
Thanks for including this section, I would have forgotten, but think it's a very good idea.